pkg/client/v1/nimbleos/ns_j_w_k_algorithm.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Hewlett Packard Enterprise Development LP
+package nimbleos
+
+// Golang package for NsJWKAlgorithm Enum.
```

Comment:
It seems appropriate to add JWK to the list of acronyms with special treatment when converting between symbol style, so that this file name is changed from ns_j_w_k_algorithm.go to ns_jwk_algorithm.go. Maybe something similar for ns_j_w_key.go too.

Reply:
Sure, I will file a separate bug for this since it has to be done in the Go Generator code.